contract/vm.go

```diff
@@ -18,10 +18,30 @@ var (
 	ErrVMNotExist = errors.New("Vm not exist in vm manager")
 )
 
+// ContextConfig define the config of context
+type ContextConfig struct {
+	XMCache      *xmodel.XMCache
+	Initiator    string
+	AuthRequire  []string
+	ContractName string
+	GasLimit     int64
+}
+
+// NewContextConfig return context config of a transaction
+func NewContextConfig(xmCache *xmodel.XMCache, initiator string, authRequire []string, contractName string, gasLimit int64) (*ContextConfig, error) {
```

Comment:
ContextConfig as a pure data object, does not require a constructor, directly fills the field with the stuct initialization syntax

Reply:
I see, thanks  for your comments.